src/types/utils.cpp

```diff
@@ -146,6 +147,29 @@ void Utils::InitializeCampbellColorTable(gsl::span<COLORREF>& table)
     // clang-format on
 }
 
+// Function Description:
+// - Fill the first 16 entries of a given color table with the Campbell color
+//   scheme, in the Windows BGR order.
+// Arguments:
+// - table: a color table with at least 16 entries
+// Return Value:
+// - <none>, throws if the table has less that 16 entries
+void Utils::InitializeCampbellColorTableForConhost(gsl::span<COLORREF>& table)
+{
+    THROW_HR_IF(E_INVALIDARG, table.size() < 16);
+    InitializeCampbellColorTable(table);
+    SwapANSIColorOrderForConhost(table);
+}
+
+void Utils::SwapANSIColorOrderForConhost(gsl::span<COLORREF>& table)

```

Comment:
Doc comment.

Reply:
And technically probably unit test...

Reply:
ooh make a cool ascii art diagram about why you've gotta do